src/main/resources/com/google/jenkins/plugins/computeengine/ui/helpers/Standard/config-detail.jelly

```diff
@@ -0,0 +1,21 @@
+<!--
+Copyright 2024 CloudBees, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+        https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form">
+    <f:entry field="maxRunDurationSeconds" title="${%Max Run Duration Seconds}">
+        <f:textbox default="0"/>
+    </f:entry>
+</j:jelly>
```

Comment:
Currently there is a duplication of configuration and html files in `spot vm` and `standard` provisioning types.
Functionality wise it is unharmful. But not good from a code maintenance standpoint. 

need to check on this duplication if possible to make it go away.

Reply:
override `Standard/config.jelly` instead that adds the maxDurationSeconds field followed by `<st:include page="config-detail.jelly" optional="false" class="${descriptor.clazz}"/>`

Reply:
thanks for ideas.. actually I could use the `j:if` and single config.jelly and single html file is sufficient.
fixed in https://github.com/jenkinsci/google-compute-engine-plugin/pull/492/commits/3e7e7872cb2989de02b2201ab92c8e9d27931281